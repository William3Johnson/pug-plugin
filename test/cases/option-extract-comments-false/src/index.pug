html
  head
    script(src=require('bootstrap/dist/js/bootstrap.min'))
  body