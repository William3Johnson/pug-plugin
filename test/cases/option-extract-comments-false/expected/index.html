<!DOCTYPE html><html><head><script src="/assets/js/vendor.dc58b31e.js"></script></head><body></body></html>